zerver/openapi/zulip.yaml

```diff
@@ -11381,6 +11396,12 @@ paths:
                               Whether other users are allowed to see whether you've
                               read messages.
 
+                              **Changes**: New in Zulip 5.0 (feature level 105).
+                          send_private_message_on_invitee_signup:
+                            type: boolean
+                            description: |
+                              Whether to get a PM when an invitee join's the realm.
+
                               **Changes**: New in Zulip 5.0 (feature level 105).
```

Comment:
This should be `**Changes**: New in Zulip 5.0 (feature level 114).` which is the latest feature level. Should be changed in other places as well.
See `templates/zerver/api/changelog.md` and `version.py` file for the last feature level.